tests/test_collection.py

```diff
@@ -24,7 +24,9 @@ def collect(code: str) -> list[FuzzProcess]:
     )
     p = Path(tempfile.mkstemp(prefix="test_", suffix=".py")[1])
     p.write_text(code)
-    fps = interface._get_hypothesis_tests_with_pytest([str(p), "-p", "no:dash"])
+    fps = interface._get_hypothesis_tests_with_pytest(
+        [str(p), "-p", "no:dash"]
```

Comment:
Oh, we can drop the `-p no:dash` now; that was to disable the plotly-dash pytest plugin.  Worth searching more generally to see if we've left it anywhere else too.

Reply:
oh thank god we can get rid of this lol